tmol/score/omega/omega_energy_term.py

```diff
@@ -0,0 +1,88 @@
+import torch
+
+from ..atom_type_dependent_term import AtomTypeDependentTerm
+from ..bond_dependent_term import BondDependentTerm
+
+from tmol.database import ParameterDatabase
+from tmol.score.common.stack_condense import tile_subset_indices
+from tmol.score.omega.params import OmegaGlobalParams
+from tmol.score.omega.omega_whole_pose_module import OmegaWholePoseScoringModule
+
+from tmol.chemical.restypes import RefinedResidueType
+from tmol.pose.packed_block_types import PackedBlockTypes
+from tmol.pose.pose_stack import PoseStack
+
+
+class OmegaEnergyTerm(AtomTypeDependentTerm, BondDependentTerm):
```

Comment:
Is this actual the correct usage of AtomTypeDependentTerm and BondDependentTerm? When are these necessary/what do they provide?